src/emu/epoc/include/epoc/services/fbs/adapter/font_adapter.h

```diff
@@ -58,6 +58,8 @@ namespace eka2l1::epoc::adapter {
         virtual void free_glyph_bitmap(std::uint8_t *data) = 0;
         virtual glyph_bitmap_type get_output_bitmap_type() const = 0;
 
+        virtual bool is_glyph_exist(std::size_t idx, std::uint32_t code) = 0;
```

Comment:
Does?

Reply:
Corrected in #34a7067000bd73b132fe4b633e7eda79a8b34a23 and #a7f47a0c13f799c920b94a3f62262072b609fe2a

Reply:
Corrected in 34a7067 and a7f47a0